src/rpc/mining.cpp

```diff
@@ -915,7 +915,11 @@ static UniValue getblocktemplate(const Config &config,
     UniValue aCaps(UniValue::VARR);
     aCaps.push_back("proposal");
 
-    Amount coinbasevalue = Amount::zero();
+    const Consensus::Params &consensusParams = chainparams.GetConsensus();
+    Amount amountFeeReward =
+        GetBlockRewardFromFees(-1 * pblocktemplate->entries[0].fees);
```

Comment:
`entries[0]`? Isn't there something attached to the template itself?

Reply:
Well, we could recalculate all nFees from entries, but it's already stored in the entry of the coinbase.

Hope we can replace this with something simpler eventually.